### PR TITLE
OPS: upgrade rnwatch

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/e5xzkvygpfn529h9t8kmzmb8t4

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/c74wa6apazcu5kng1re3um78nc

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/8zmr9w4mjeumvn01uwaah9kxac

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/fn1599cwdgx848r2y1y1e382qw

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/kdev2uxn09zqh973vj7kdmc25m

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/u6vzxpmucm9n4kjg0tbvp9y8ug

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/nm3ggmjx7z4fwqh40zhuyxrj38

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/rjvjckpdz2gdh5bdazhq0dbey4

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/5thp1e8x1m3uep6gxddqdp1mh8

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/3a7x8t2aap52jp32ya34xnbg24

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/wzpgq5qn84vezrduu0rfy85rv4

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/9wqkb1n07udxq75m9x3h2xmg98

File: ios/BlueWallet/Info.plist [hunk 1]
```diff
@@ -47,6 +47,21 @@
 		<true/>
 		<key>NSExceptionDomains</key>
 		<dict>
+			<key>electrum3.bluewallet.io</key>
```
Comment (Overtorment) [inline ios/BlueWallet/Info.plist]: do we need it here? first, we are supposed to have unencrypted HTTP for all (we made it for custom LNDHUB). second, electrum is not http or https, its pure TCP

**Pull request merged**